tests/runtime/int.ts

```diff
@@ -0,0 +1,56 @@
+export function toBytesLE(value: bigint): Uint8Array {
```

Comment:
This file is an awesome candidate for some unit tests.

Reply:
Agreed. It feels weird to unit test the unit tests though :stuck_out_tongue:.